adminSiteClient/GrapherConfigGridEditorTypesAndUtils.tsx

```diff
@@ -293,6 +321,134 @@ export function getEqualityOperator(str: string): EqualityOperator | undefined {
     else return undefined
 }
 
+/** JsonLogic is the easiest format that the React Awesome Query Builder can round
+    trip (i.e. deserialize from). Building the internal structure of the query library
+    would be tedious so we convert our SExpressions to JsonLogic. */
+export function SExpressionToJsonLogic(
```

Comment:
This one is very well-suited to be written as a class method of the different `Operation`s, don't you think?
I.e. add it alongside `toSql()`, `toSExpr()`.